code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -0,0 +1,475 @@
+/datum/species/psyphoza
+	name = "\improper Psyphoza"
+	plural_form = "Psyphoza"
+	id = SPECIES_PSYPHOZA
+	bodyflag = FLAG_PSYPHOZA
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/psyphoza
+	species_traits = list(NOEYESPRITES, AGENDER, MUTCOLORS)
+	sexes = FALSE
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP
+	species_language_holder = /datum/language_holder/psyphoza
+	exotic_blood = /datum/reagent/drug/mushroomhallucinogen
+	allow_numbers_in_name = TRUE
+	inert_mutation = SPORES
+
+	offset_features = list(OFFSET_UNIFORM = list(0,0), OFFSET_ID = list(0,0), OFFSET_GLOVES = list(0,0), OFFSET_GLASSES = list(0,-2), OFFSET_EARS = list(0,-3), OFFSET_SHOES = list(0,0), OFFSET_S_STORE = list(0,0), OFFSET_FACEMASK = list(0,-2), OFFSET_HEAD = list(0,-2), OFFSET_FACE = list(0,-2), OFFSET_BELT = list(0,0), OFFSET_BACK = list(0,0), OFFSET_SUIT = list(0,0), OFFSET_NECK = list(0,0))
+
+	mutant_brain = /obj/item/organ/brain/psyphoza
+	mutanteyes = /obj/item/organ/eyes/psyphoza
+	mutanttongue = /obj/item/organ/tongue/psyphoza
+
+	mutant_bodyparts = list("psyphoza_cap")
+	default_features = list("psyphoza_cap" = "Portobello", "body_size" = "Normal", "mcolor" = "fff")
+	hair_color = "fixedmutcolor"
+
+	species_chest = /obj/item/bodypart/chest/psyphoza
+	species_head = /obj/item/bodypart/head/psyphoza
+	species_l_arm = /obj/item/bodypart/l_arm/psyphoza
+	species_r_arm = /obj/item/bodypart/r_arm/psyphoza
+	species_l_leg = /obj/item/bodypart/l_leg/psyphoza
+	species_r_leg = /obj/item/bodypart/r_leg/psyphoza
+
+	//Fire bad!
+	burnmod = 1.25
+
+	//Reference to psychic highlight action
+	var/datum/action/item_action/organ_action/psychic_highlight/PH
+
+/datum/species/psyphoza/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	. = ..()
+	PH = locate(/datum/action/item_action/organ_action/psychic_highlight) in C.actions
+	ADD_TRAIT(C, TRAIT_PSYCHIC_SENSE, SPECIES_TRAIT)
```

Comment:
does transforming and un-transforming someone to a psyphoza work as expected?

Reply:
Whoops, add ``REMOVE_TRAIT`` to species loss.